llama-index-core/llama_index/core/indices/vector_store/base.py

```diff
@@ -299,9 +301,8 @@ def build_index_from_nodes(
         if any(
             node.get_content(metadata_mode=MetadataMode.EMBED) == "" for node in nodes
         ):
-            raise ValueError(
-                "Cannot build index from nodes with no content. "
-                "Please ensure all nodes have content."
+            raise warnings.warn(
```

Comment:
I don't think it's possible to raise a warning? I think we need to
1. Report if any inputs have no content
2. filter them out of the `nodes` list before calling `self._build_index_from_nodes()`
3. Lets add a unit test for this